draft-ietf-scitt-architecture.md

```diff
@@ -619,6 +619,40 @@ For example, consider a Transparency Service implemented using a set of replicas
 Each replica MAY provide a recent attestation report for its TEE, binding their hardware platform to the software that runs the Transparency Service, the long-term public key of the service, and the key used by the replica for signing Receipts.
 This attestation evidence can be supplemented with Receipts for the software and configuration of the service, as measured in its attestation report.
 
+### Configuration
```

Comment:
To make life simple, we should only keep Registration Information as the Element which needs updating, hence using the generic term Configuration, we should adhere to a more specific term known as 
`Registration Policy` which is an updatable element.

Reply:
This was a great discussion during the hackathon. We talked about having multiple statements for configuration, and registration policies allowing a service to put RBAC on who could read the files. We likely needed an aggregated statement to reference the different config and registration policies. This way the receipt would need only one. This is additional content we wanted to follow-up with after this baseline as it introduces more details we wanted to incrementally think through.

Reply:
@yogeshbdeshpande, are you ok with deferring the additional detail? Please 👍 if you agree.

Reply:
@yogeshbdeshpande, please open a new issue/pr if you feel there is missing info.